README.md

```diff
@@ -258,16 +264,12 @@ Fill in the details required for this post build action. You need the following
 
 You need to now enter the credentials that lets Jenkins trigger a release with the latest completed build, on your behalf. If you are using VS Team Services, you just need to enter **PAT** with atleast "Release (read, write and execute)" scope. (Refer to this [link](https://www.visualstudio.com/en-us/get-started/setup/use-personal-access-tokens-to-authenticate) to understand how to create PAT). In case you are using TFS, you need to enter the **username** and **password**.
```

Comment:
> with atleast

`with at least`

> In case you are using TFS, you need to enter the **username** and **password**.

If only Visual Studio Team Services are supported, you could omit TFS-specific instructions.